.github/workflows/npm_package_pr_npm.yaml

```diff
@@ -51,7 +51,7 @@ jobs:
       - run: npm run validate:ci
 
       # Sanity check so we don't end up with bad builds in main
-      - run: npm ci --ignore-scripts --omit=dev --omit=peer
+      - run: npm ci --ignore-scripts --omit=dev
```

Comment:
There is no reason to allow build scripts or dev deps. So continuing to omit those